categories.py

```diff
@@ -100,6 +100,26 @@
   "cseez":            ["hgcal-dpg"],
   "fabiocos":         ["mtd-dpg", "operations"],
   "parbol":           ["mtd-dpg"],
+  # pogs
+  "juska":            ["pf"],
+  "kdlong":           ["pf"],
+  "lfinco":           ["egamma-pog"],
+  "swagata87":        ["egamma-pog"],
+  "kirschen":         ["jetmet-pog"],
+  #"MC LEAN, Christine Angela": ["jetmet-pog"],
+  "knollejo":         ["lumi-pog"],
+  #"SCHWICK, Christoph":         ["lumi-pog"],
+  "gkaratha":         ["muon-pog"],
+  #"SANCHEZ CRUZ, Sergio":         ["muon-pog"],
+  "johnalison":       ["btv-pog"],
+  "soureek":          ["btv-pog"],
+  "michael-pitt":     ["proton-pog"],
+  "kshcheli":         ["proton-pog"],
+  "kandrosov":        ["tau-pog"],
+  #"LE BIHAN, Anne-Catherine":        ["tau-pog"],
+  "mmusich":          ["tracking-pog"],
```

Comment:
@srimanob , this will override https://github.com/cms-sw/cms-bot/blob/master/categories.py#L82 .  please remove this line and add `tracking-pog` at https://github.com/cms-sw/cms-bot/blob/master/categories.py#L82